src/clr-icons/shapes/essential-shapes.ts

```diff
@@ -813,7 +827,14 @@ export const EssentialShapes: any = {
     "accessibility-2": ClrShapeAccessibility2,
     "sort-by": ClrShapeSortBy,
     "collapse-card": ClrShapeCollapseCard,
-    "expand-card": ClrShapeExpandCard
+    "expand-card": ClrShapeExpandCard,
+    "briefcase": ClrShapeBriefcase,
+    "color-picker": ClrShapeColorPicker,
+    "copy-to-clipboard": ClrShapeCopyToClipboard,
+    "filter-2": ClrShapeFilter2,
```

Comment:
Why `filter-2`? Are we adding icons like `name-1`, `name-2` etc?

Reply:
we have `accessibility-1` and `accessibility-2`.

Reply:
This icon is a more modern update to the old style of filter we currently have. We have some other instances that called for a second version. name-1, name-2 is the convention we have been using so far.